### PR TITLE
Added survey service config to rops

### PR DESCRIPTION
**Pull request by madeye-matt**
This is needed for docker images to work post us101 changes to survey service

### CODE REVIEW

**Comment by gemmai95** [comment]
Does survey service need the database migration config?

**Comment by madeye-matt** [comment]
@gemmai95 nope, the Dockerfile makes sure the migration scripts are put in the default location on the docker image (/db-migrations)

File: ras-services.yml [hunk 1]
```diff
@@ -244,6 +244,9 @@ services:
       - 8085:8085
     environment:
       - BACKSTAGE_API_URL=http://${BACKSTAGE_HOST}:${BACKSTAGE_PORT}/backstage-api
+      - SURVEY_URL=http://${SURVEY_HOST}:${SURVEY_PORT}
+      - SURVEY_USERNAME=admin
```
Comment (tej99) [inline ras-services.yml]: SURVEY_USERNAME =${SECURITY_USER_NAME}?
SURVEY_PASSWORD =${SECURITY_USER_PASSWORD}?

**Comment by benjefferies** [comment]
@madeye-matt I think we'll need to add it to https://github.com/ONSdigital/ras-deploy/ too

**Comment by madeye-matt** [comment]
@tej99 done
@benjefferies will do! Although I can't come sit with you when it all goes bang as I am WFH ;-)

**Pull request merged**